UltimateTeam.Toolkit/Models/AuctionResponse.cs

```diff
@@ -2,13 +2,13 @@
 
 namespace UltimateTeam.Toolkit.Models
 {
-    public class AuctionResponse
+    public class AuctionResponse : IListResponse
```

Comment:
Would be great if you could push another commit to your branch that removes ´virtual`.